product_docs/docs/biganimal/release/overview/deployment_options/postgres_admin_user.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: BigAnimal's cloud account ba_admin_user
```

Comment:
"BigAnimal's cloud account" is a strange way to describe this.

Reply:
I thought these builds, and therefore, the user, only applied to BAH. Is that not the case?

Reply:
That's a separate question (it will go to BYOA as well). My concern is more about using the phrase "BigAnimal's cloud account" itself.

Reply:
My first draft of this content assumed that the builds and new role and user only applied to BAH. Hence the confusion. I'm about to commit a new version that doesn't have a need to refer to BAH.